cmd/notation/sign.go

```diff
@@ -79,7 +83,7 @@ func runSign(command *cobra.Command, cmdOpts *signOpts) error {
 	if err != nil {
 		return err
 	}
-	_, err = notation.Sign(command.Context(), signer, sigRepo, opts)
+	_, err = notation.Sign(ctx, signer, sigRepo, opts)
```

Comment:
Not sure if replacing command.Context() directly with ctx would cause any issue? An alternative might be passing command.Context() into cmdOpts.LoggingFlagOpts.SetLoggerLevel().

Reply:
Updated.